modelling/src/main/java/org/axonframework/modelling/HierarchicalStateManagerConfigurationEnhancer.java

```diff
@@ -45,12 +45,6 @@ public int order() {
 
     @Override
     public void enhance(@Nonnull ComponentRegistry componentRegistry) {
-        if (!componentRegistry.hasComponent(StateManager.class)) {
```

Comment:
Curious: why is this empty version no longer required for the hierarchical one?

Reply:
If we by default register a `StateManager` on all configuration levels, that means modules not declaring entities also get one. 

So, with the StatefulCommandHandlingModule having EntityModules, these entity modules also get a StateManager, and so they register it to.. their own. Not to their parent. So it was not possible to access the entity at all.

In addition, this makes `StateManagers` where not necessary, so this was removed.

Reply:
Alright, thanks for clarifying!